ckan/plugins/core.py

```diff
@@ -136,20 +128,21 @@ def plugins_update() -> None:
     environment.update_config()
 
 
-def load_all() -> None:
+def load_all(force_update: bool = False) -> None:
     '''
     Load all plugins listed in the 'ckan.plugins' config directive.
     '''
     # Clear any loaded plugins
     unload_all()
 
-    plugins = aslist(config.get('ckan.plugins')) + find_system_plugins()
+    plugins = aslist(config.get('ckan.plugins'))
 
-    load(*plugins)
+    load(*plugins, force_update=force_update)
 
 
 def load(
-        *plugins: str
+        *plugins: str,
+        force_update: bool = False,
 ) -> Plugin | list[Plugin]:
```

Comment:
It would be nice to document here what force_update is and why we needed.

Reply:
added docstrings to `load` and `load_all`